pkg/apis/pipelines/v1alpha1/gitopsservice_types.go

```diff
@@ -12,6 +12,7 @@ type GitopsServiceSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
```

Comment:
Remove this comment, and add a proper comment for the Prefix field, it's embedded in the CRD.

https://github.com/redhat-developer/gitops-operator/pull/10/commits/a27b74582dd4544e860b31eb936a9c197e3c2018#diff-4284bda429d34d9eb75be67605112b07f66752910ea60847e9980bec6b9afccfR35